src/main/java/ch/njol/skript/lang/SkriptParser.java

```diff
@@ -577,6 +582,12 @@ public boolean hasTag(String tag) {
 				log.printError();
 				return null;
 			}
+			if (expr.contains("(") && expr.endsWith(")") && expr.indexOf("(") < expr.indexOf(")")) {
```

Comment:
contains is made unnecessary by indexOf, and endsWith should always be checked first for efficiency